powersimdata/input/tests/test_change_table.py

```diff
@@ -492,3 +510,66 @@ def test_remove_bus(ct):
         ct.remove_bus({845})
     ct.scale_plant_capacity(resource="ng", plant_id={0: 0})
     ct.remove_bus({845})
+
+
+def test_add_demand_flexibility(ct):
+    with pytest.raises(Exception):
```

Comment:
This would raise `ValueError` I think? If so we could swap `Exception` for that - usually good to pick the most specific one.

Reply:
Oh okay. I can try to find the more specific error that will be raised for each of those.